scripts/validate_docstrings.py

```diff
@@ -127,7 +126,12 @@ def doc_parameters(self):
 
     @property
     def signature_parameters(self):
-        if not inspect.isfunction(self.method_obj):
+        if not (inspect.isfunction(self.method_obj)
+                or inspect.isclass(self.method_obj)):
+            return tuple()
+        if (inspect.isclass(self.method_obj)
+                and self.method_name.split('.')[-1] in {'dt', 'str', 'cat'}):
```

Comment:
Is it difficult to find the accessors that have been registered, instead of hardcoding them here? So we don't need to remember to change this when a new accessor is created.

Reply:
Yeah, I was looking into the class (eg that I could check they inherit from a base Accessor class, but that's not consistent between them). 
So not really sure there is a better way, but totally agree it is not ideal.

Reply:
Actually, there is a `_accessors` attribute. But not fully sure to what extent we can rely on that.

Reply:
The problem is then that I need to get the class (eg Series in case of pandas.Series.dt), which is also difficult to introspect. So maybe the easiest is to leave it as is.

Reply:
Sure, it's quite useful as it is, we can improve it in the future if we find a way.

Reply:
FYI: https://github.com/pandas-dev/pandas/issues/19963

I think `isinstance(object.__getattribute__(self, method_obj.__name__), CachedAccessor)` might have done it, But having an official registry is better :)